CHANGELOG.rst

```diff
@@ -1,6 +1,16 @@
 Changelog
 =========
 
+Changes in Version 1.1.0
+------------------------
+
+- Use ``boto3`` to fetch on-demand credentials, expanding and standardizing authorization capabilities.
```

Comment:
Can we mention the newly support EKS feature?

Reply:
Done